templates/dashboard/order/modal/remove_order.html

```diff
@@ -0,0 +1,30 @@
+{% extends "dashboard/base_modal.html" %}
+
+{% load i18n %}
+{% load materializecss %}
+
+{% block form_class %}{% endblock %}
+
+{% block action_url %}
+{% url 'dashboard:draft-order-delete' order_pk=order.pk %}
+{% endblock %}
+
+{% block title %}
+{% trans "Remove order" context "Modal remove order title" %}
```

Comment:
Same here. Maybe "Delete draft order".

Reply:
You are right, it will be better - I will change it.